tests/clojure/initialize_repl_with_script.exp

```diff
@@ -6,15 +6,15 @@ set _ "(?:.*?)"
 set timeout -1
 spawn ./prybar-clojure -i ./test_files/hello.clj
 match_max 100000
-expect -re "^${_}hello${_}\r
-${_}Clojure${_}\r
-${_}--> $"
+expect -re "^${_}Clojure${_}\r
```

Comment:
Thank you for patching the tests to work with your changes!

When I run `make test-image`, the tests fail for me. It may be something silly! Can you try running the tests manually inside the docker image?

```
clojure:
...     run code
========== "clojure/run_code.exp" failed ==========

make: *** [test-image] Error 1
```

Reply:
Interesting... I tried both `make test-image` and `make test` (inside the container) and the tests are all passing... 🤔

I can see you got an error right at the first Clojure test. One possibility is that it might be timing out while downloading the Clojure JARs when `prybar-clojure` was run for the first time (very likely on a slow internet connection). This is in turn due to the `clojure` command that lazily downloads the Clojure JARs when it's run for the first time after a fresh installation. I can go ahead and add a line right after the installation steps in the `scripts/docker-install.sh` so that we can trigger the download eagerly during container image creation. The code there should look something like this afterwards:
```sh
clj_cli_version=1.10.1.536
wget "https://download.clojure.org/install/linux-install-${clj_cli_version}.sh"
chmod +x "linux-install-${clj_cli_version}.sh"
"./linux-install-${clj_cli_version}.sh"
# Eagerly download Clojure JARs during container image creation.
clojure -Sverbose -Sforce --eval ""
```

Reply:
Aha! Yes, please, let's eagerly fetch those jars.